leetcodePython/BinarySearch/bs_1146.py

```diff
@@ -0,0 +1,45 @@
+import bisect
+
+
+class SnapshotArray_BruteForce:
+    def __init__(self, length: int):
+        self._snap_id = -1
+        self._array = [0] * length
```

Comment:
This copy is only valid for primitive data init. If you do the same for array or reference data, it will become reference copy